### DISCUSSION TITLE
Is that using "server" folder in production will auto generate a subdomain with pattern like "backend.rootdomain.xyz"?

### DISCUSSION THREAD

**Discussion by jdkcoder** [nuxt/nuxt#25898] [Questions]
I'm using "server" folder to make a proxy layer via my backend, sometimes, when error comes, instead of "rootdomain.xyz/api/auth/login" I saw in console is "backend.rootdomain.xyz" like this image below. Is there anyone met this before?

![image](https://github.com/nuxt/nuxt/assets/89140442/7eeb0b52-e2de-4108-a5b7-ff9acc595762)

**Comment by TheAlexLichter** [comment]
Shouldn't be related to nuxt, no.
